feature/search/src/main/java/com/terning/feature/search/search/SearchRoute.kt

```diff
@@ -67,6 +69,11 @@ fun SearchRoute(
             }
     }
 
+    val bannerList = when (bannerState.searchBannersList) {
+        is UiState.Success -> (bannerState.searchBannersList as UiState.Success<List<com.terning.domain.search.entity.SearchBanner>>).data.toImmutableList()
+        else -> emptyList()
+    }
+
```

Comment:
흠 그리고 feature에서 domain의 데이터클래스를 사용하는 거에 대해 괜찮을지에 대한 의문이 드네요..
feature에서 사용할 모델을 따로 만들어 주는 건 어떤가요.....?


```suggestion
    val bannerList = when (bannerState.searchBannersList) {
        is UiState.Success -> (bannerState.searchBannersList as UiState.Success<List<SearchBanner>>).data.toImmutableList()
        else -> emptyList()
    }
```

Reply:
헉 감사해욤 feature에 있는 모델 State로 변경했습니다